deployments/CRD/KubeArmorPolicy.yaml

```diff
@@ -358,8 +361,6 @@ spec:
                           items:
                             type: string
                           type: array
-                      required:
-                      - path
```

Comment:
Ideally we want either path/bin to be required. 
CRDs do tend to support it through `oneOf` concept but kubebuilder has no API to create that validation